cmd/gcp-controller-manager/main.go

```diff
@@ -77,6 +77,7 @@ var (
 	hmsSyncNodeURL                     = pflag.String("hms-sync-node-url", "", "URL for reaching the Hosted Master Service SyncNode API.")
 	kubeletReadOnlyCSRApprover         = pflag.Bool("kubelet-read-only-csr-approver", false, "Enable kubelet readonly csr approver or not")
 	autopilotEnabled                   = pflag.Bool("autopilot", false, "Is this a GKE Autopilot cluster.")
+	clearStalePodsOnNodeRegistration   = pflag.Bool("clearStalePodsOnNodeRegistration", false, "If true, after preemption, clean pods bound to old node.")
```

Comment:
nit: `If true, after node registration, clean pods bound to old node.`

Reply:
should I wait for this to be fixed, before I approve? If I approve this will be automerged.

Reply:
Done changing.